java/ql/test/library-tests/frameworks/guava/flow.ql

```diff
@@ -0,0 +1,16 @@
+import java
```

Comment:
Could you refine this query to reduce the noise in the expected output. Having clearer expected output makes it easier to review it for correctness.